app/sign-up.tsx

```diff
@@ -92,19 +99,31 @@ export default function SignUpPage() {
           <Text className="text-red-500 mb-2 text-center">Passwords do not match</Text>
         )}
 
-        <TouchableOpacity
-          className="w-full bg-blue-500 p-2 rounded"
-          onPress={handleSubmit(onSubmit)}
-          disabled={isLoading}
-        >
-          <Text className="text-white text-center">
-            {isLoading ? 'Signing up...' : 'Sign Up'}
-          </Text>
-        </TouchableOpacity>
+        {signUpMessage ? (
+          <View className="mb-4">
+            <Text className="text-blue-500 font-bold font-lg text-center mb-2">{signUpMessage}</Text>
+            <TouchableOpacity
```

Comment:
I saw a warning in the console today that TouchableOpacity is going to be deprecated and replaced by Pressable in the future. Let's use this instead, unless there's a good reason we shouldn't (I'm aware we'll have a lot of them to change in the codebase as Claude chose to use them).

see https://reactnative.dev/docs/touchableopacity

Reply:
Yeah I've seen a few similar warnings about deprecated elements, shall I work through all of them?